fixtures/User.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace Nelmio\Alice;
+
+/**
+ * @package Nelmio\Alice
+ */
+class User
+{
+    private $name;
```

Comment:
since `$name` is not set in a constructor I think it's a good practice to add a phpdoc so here `@var string|null`